Makefile

```diff
@@ -2,6 +2,46 @@
 help:
 	@cat Makefile | grep '^## ' --color=never | cut -c4- | sed -e "`printf 's/ - /\t- /;'`" | column -s "`printf '\t'`" -t
 
+## build - Build the project in Release mode
+build:
+	dotnet msbuild -property:Configuration="Debug" -target:Rebuild -restore
```

Comment:
Debug and Release are flipped

Reply:
The config itself was intentional. The command `build` should be the generic debug since people would be running that much more than a production build which should require a separate target; however, the comments are definitely off. Fixed.